pkg/templates/aws/resources/subnet.yaml

```diff
@@ -54,6 +54,11 @@ properties:
   MapPublicIpOnLaunch:
     type: bool
     default_value: false
+  Id:
+    type: string
+    configuration_disabled: true
+    deploy_time: true
+    required: true
```

Comment:
What does `deploy_time` + `required` mean? Seems paradoxical except for imports. I'm leaning to a different flag than `required` to indicate which property is the unique identifier - this will be needed to distinguish which property to use when doing the IaC when there are multiple required properties.

Reply:
right now it means required for importing since we know when we create a resource, we wont have the deploy_time properties.

Not entirely sure what you mean about distinguishing for iac